frontend/src/components/Input/index.js

```diff
@@ -2,7 +2,16 @@ import { PropTypes } from "prop-types";
 import styles from "./index.module.css";
 import classNames from "classnames";
 
-const Input = ({ type, placeholder, disabled, error, errorMessage }) => {
+const Input = ({
+  id,
+  type,
+  placeholder,
+  disabled,
+  error,
+  errorMessage,
+  value,
```

Comment:
Why is `value` a prop? Is the parent passing down this?

Reply:
The parent will. My idea is that the signup page will have `const [name, setName] = useState("")` and I'll pass value={name} and onChange={(event) => setName(event.target.value)}.

Reply:
Are you trying to pass the `value` back to the parent? [Lifting state up](https://reactjs.org/docs/lifting-state-up.html)?

Reply:
Then the same for email I assume

Reply:
I decided to push this first before the Signup page to build on a branch created out of this branch

Reply:
Yes I think I am (only saw that message now after updating the page )

Reply:
Did you try this?

Reply:
Yes, and it worked. Maximino helped me. See [thread](https://subvisualuniverse.slack.com/archives/C03KFP6GQR3/p1659695443983429)

Reply:
Ah, I see!

Reply:
Got it.